packages/redux/src/contents/serverInitialState.ts

```diff
@@ -15,28 +18,36 @@ import type { ServerInitialState } from '../types/serverInitialState.types.js';
  *
  * @returns Initial state for the contents reducer.
  */
-const serverInitialState: ServerInitialState = ({ model }) => {
+const serverInitialState: ServerInitialState = ({ model, strategy }) => {
   if (!get(model, 'searchContentRequests')) {
     return { contents: INITIAL_STATE_CONTENT };
   }
 
   const { searchContentRequests, slug, seoMetadata, subfolder } = model;
+  const url = subfolder !== '/' ? slug?.replace(subfolder, '') : slug;
+  const normalizedUrl = url
```

Comment:
Is this necessary? If json=true is passed, the response returned by FO will not be html, so the application will not be run in this case.